### ISSUE TITLE
Hi, are you interested in expanding the data?

### ISSUE THREAD

**Issue by TK3600** [ResidentMario/wargame#1]
I am a very experienced modder who has discovered quite a few new things. Do you accept further revisions?

**Comment by ResidentMario** [comment]
Yeah I'd be down to update this reference, I know a lot of people have found it helpful over the years.

**Comment by TK3600** [comment]
Lets start with AI; I am specialized in that area.

ai module

comportementIAstrat (AI unit behaviour)

1 arty
2 close range arty? burritino special??
3 tanks, inf; main attacking units
4 SAM
5 command
6 spaag
7 recon
8 supply convoy? (navy dont look supplying) (HEMTT)
9 unarmed transport
10 ASF
11 Bomber/unarmed strike plane
12 multi-role
13 SEAD
14 flagships
15 AshM ships
16 river boats

typeforpoolselection (How AI prioritize use it)

1 (missin ADATS AI) [null] [referenced in ai]
2 unarmed ATGM attack plane
3 basic transport
4 ATGM heli
5 AT+AA heli
6 SAM
7 SPAAG
8 ASF
9 AShM plane
10 naval heli
11 Ashm land launcher
12 tracked CV car
13 armored supply vehicle
14 tank CV, no shit given to any threat
15 recon combat vehicle
16 tube arty
17 (missing RR vehicle) [null]
18 (missing cl vehicle type referenced in AI) (anti vehicle?) [null]
19 naval transport
20 unarmed bomber/napalm
21 calvry tank
22 [null] (missing cluster bomber)
23 heli CV
24 car CV, run away from danger
25 inf SF
26 recon coastal support
27 arty boat
28 [null] DEPR
29 inf milia recon
30 [null] (missing elite infantry)
31 CIWS+missle combat escort
32 Air defense corvette
33 CIWS patrol boat
34 ASM boat
35 FIST
36 command ship
37 coaster CV chip
38 flame tanks
39 gun+aa helo
40 gunship
41 IFV, ignores some threat; somewhat suicidal.
42 [null] (missing support infantry type)
43 inf line
44 MANPADS
45 inf atgm
46 inf CV, only hide in specific spots? Dont move.
47 inf combat recon
48 interceptors (tomcat/mig31m)
49 light inf
50 Basic tanks
51 expensive tanks 85+
52 Medium tanks 45-85
53 MLRS
54 Militias
55 Mortar
56 multirole plane; 
57 naval supply ship
58 recon heli
59 recon car
60 SEAD
61 inf sf recon
62 supply trucks
63 FOB
64 supply helli
65 fire support vehicle
66 Fodders/FSV (tank destroyer)
67 transport heli
68 inf flamer
